specification.md

```diff
@@ -424,7 +424,7 @@ We can use an action data filter to filter only the breads data:
     {  
        "functionRef": "breadAndPastaTypesFunction",
        "actionDataFilter": {
-          "results": "${ {breads: .breads} }"
+          "results": "${breads: .breads}"
```

Comment:
{breads: .breads}  is a jq expression
"breads: .breads" is not

Reply:
same again